build/gulpfile.js

```diff
@@ -56,12 +58,17 @@ module.exports = function(gulp, config, pluginName){
         module: 'commonjs',
         declarationFiles: true,
         noExternalResolve: false,
-        removeComments: true
-      })))
-      .on('error', plugins.notify.onError({
```

Comment:
that just outputs to console though, right? no *toast* notification messages pop?